quantum/keymap_extras/keymap_swedish_osx_iso.h

```diff
@@ -0,0 +1,240 @@
+// Row 3
+#define SE_A    KC_A    // A
+#define SE_S    KC_S    // S
+#define SE_D    KC_D    // D
+#define SE_F    KC_F    // F
+#define SE_G    KC_G    // G
+#define SE_H    KC_H    // H
+#define SE_J    KC_J    // J
+#define SE_K    KC_K    // K
+#define SE_L    KC_L    // L
+#define SE_ODIA KC_SCLN // Ö
+#define SE_ADIA KC_QUOT // Ä
+#define SE_QUOT KC_NUHS // '
+// Row 4
+#define SE_LABK KC_NUBS // <
+#define SE_Z    KC_Z    // Z
+#define SE_X    KC_X    // X
+#define SE_C    KC_C    // C
+#define SE_V    KC_V    // V
+#define SE_B    KC_B    // B
+#define SE_N    KC_N    // N
+#define SE_M    KC_M    // M
+#define SE_COMM KC_COMM // ,
+#define SE_DOT  KC_DOT  // .
+#define SE_MINS KC_SLSH // -
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ° │ ! │ " │ # │ € │ % │ & │ / │ ( │ ) │ = │ ? │ ` │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │   │   │   │   │   │   │   │ ^ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │   │   │   │   │   │   │ * │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ > │   │   │   │   │   │   │   │ ; │ : │ _ │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define SE_DEG S(SE_SECT)  // °
+#define SE_EXLM S(SE_1)    // !
+#define SE_DQUO S(SE_2)    // "
+#define SE_HASH S(SE_3)    // #
+#define SE_CURR S(SE_4)    // €
+#define SE_PERC S(SE_5)    // %
+#define SE_AMPR S(SE_6)    // &
+#define SE_SLSH S(SE_7)    // /
+#define SE_LPRN S(SE_8)    // (
+#define SE_RPRN S(SE_9)    // )
+#define SE_EQL  S(SE_0)    // =
+#define SE_QUES S(SE_PLUS) // ?
+#define SE_GRV  S(SE_ACUT) // `
+// Row 2
+#define SE_CIRC S(SE_DIAE) // ^ (dead)
+// Row 3
+#define SE_ASTR S(SE_QUOT) // *
+// Row 4
+#define SE_RABK S(SE_LABK) // >
+#define SE_SCLN S(SE_COMM) // ;
+#define SE_COLN S(SE_DOT)  // :
+#define SE_UNDS S(SE_MINS) // _
+
+/* Alted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ¶ │ © │ @ │ £ │ $ │ ∞ │   │ | │ [ │ ] │ ≈ │ ± │   │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ • │ Ω │ é │ ® │ † │ µ │ ü │ ı │ œ │ π │ ˙ │ ~ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │  │ ß │ ∂ │ ƒ │ ¸ │ ˛ │ √ │ ª │ ﬁ │ ø │ æ │ ™ │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ ≤ │ ÷ │   │ ç │ ‹ │ › │ ‘ │ ’ │ ‚ │ … │ – │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1 
+#define SE_PILC A(SE_SECT) // ¶
+#define SE_COPY A(SE_1)    // ©
+#define SE_AT   A(SE_2)    // @
```

Comment:
Maybe I'm missunderstanding something but I think that this is correct.

This is a picture of my swedish macbook keyboard and you send an @ with `A(SE_2)` ![IMG_2496](https://user-images.githubusercontent.com/7823236/124229350-c0c31400-db0d-11eb-8d7a-361ec19445f7.jpeg)

Reply:
That is the `Swedish - Pro` layout, which I assume is a special arrangement only for Macs (or perhaps the Macbook Pro specifically, given the name). According to [this SE question](https://apple.stackexchange.com/questions/60831/whats-the-difference-between-the-swedish-and-swedish-pro-keyboard-layouts), the only difference is apparently the swapped @ and ™.

Reply:
I would like to keep the the files as they are so that they match the `Swedish - Pro` layout. 

I've never used anything else, and I really believe that it comes as the default setting on swedish macs. Otherwise the mac keyboards doesn't really work with the labels on the keys.

And if @ and ™ must be swapped the keycodes will not match the actual keys printed, and that was the origin story behind this PR. 🙂

Reply:
That's fine, as long as you do so in a separate set of headers eg. `keymap_swedish_pro_osx_[ansi|iso].h`.

Reply:
It's also worth noting that these headers are kinda useless if you're typing on a Mac keyboard - they don't run QMK. Which is why I'm not terribly concerned that they don't match up.

Reply:
> That's fine, as long as you do so in a separate set of headers eg. keymap_swedish_pro_osx_[ansi|iso].h.

Great I'll fix more files.

> It's also worth noting that these headers are kinda useless if you're typing on a Mac keyboard - they don't run QMK. Which is why I'm not terribly concerned that they don't match up.

I don't understand how you mean. The `Swedish - Pro` layout affects QMK keyboards? Wouldn't I have to specifically set the QMK keyboard to the legacy `Swedish` layout to get `@` working correctly otherwise?

Reply:
The defines in these headers, on their own, don't really do a whole lot. They're just aliases to the "US ANSI" keycodes as defined in the HID spec. If you add `SE_ODIA` to your keymap and press it, QMK isn't sending an `ö`, it sends the same value as it would for `KC_SCLN`, and only if your OS is set up to use the Swedish keyboard layout will you get an `ö`. So you can think of them more as keycaps; they just help with readability in your `keymap.c`.

QMK obviously doesn't run on builtin Macbook keyboards, or other Apple-branded keyboards. Since this `Swedish - Pro` layout appears to be Macbook Pro-specific, it warrants an additional set of files if someone wants to use it on an external keyboard. Otherwise, my assumption was that users would simply pick the normal Swedish layout from the Input Sources list.